### PR TITLE
Transfer : verify_checksum is not properly passed to FTS + py3k fix . Closes #1040

### LINKED ISSUE #1040: Transfer : verify_checksum is not properly passed to FTS
Motivation
----------
The verify_checksum RSE attribute is passed as a string instead of a bool to FTS

Modification
------------
Cast the string to bool

### PR DESCRIPTION
**Pull request by cserf**
Transfer : verify_checksum is not properly passed to FTS + py3k fix . Closes #1040

### CODE REVIEW

File: lib/rucio/core/transfer.py [hunk 1]
```diff
@@ -677,7 +677,7 @@ def get_transfer_requests_and_source_replicas(process=None, total_processes=None
                                  'filesize': bytes,
                                  'md5': md5,
                                  'adler32': adler32,
-                                 'verify_checksum': rse_attrs[dest_rse_id].get('verify_checksum', True)}
+                                 'verify_checksum': json.loads(rse_attrs[dest_rse_id].get('verify_checksum', 'True').lower())}
```
Comment (bari12) [inline lib/rucio/core/transfer.py:680]: I am not sure if this works? For the default (if it is not set, it is fine) - but if verify_checksum is set to `True` or `False` I don't think this works.

Comment (cserf) [reply lib/rucio/core/transfer.py:680]: I tested it on integration nodes and it works ;-)

Comment (mlassnig) [reply lib/rucio/core/transfer.py:680]: wouldn't it suffice to cast it to bool() instead of running it through the json parser?

Comment (mlassnig) [reply lib/rucio/core/transfer.py:680]: ok, this is confusing as hell, apparently the json.loads really is the best variant to do this. bool() does not do what i expected it to do.

**Review by mlassnig — APPROVED** [review]

**Pull request merged**